eox_core/edxapp_wrapper/backends/users_h_v1.py

```diff
@@ -7,9 +7,11 @@
 
```

Comment:
Are we 100% compatible with this implementation with the old one?

If we are not, then we should add a new `users_h_v2` backend and do the implementation there.
If we are, we can keep things here but we must be careful that we dont break the compatibility